packages/http-client/src/__tests__/remote-admin-api.test.ts

```diff
@@ -95,7 +86,7 @@ test('stopIndexingModels()', async () => {
   await adminApi.stopIndexingModels([MODEL])
 
   expect(fauxFetch.mock.calls[0][0]).toEqual(new URL(`https://example.com/admin/getCode`))
-  expect(fauxFetch.mock.calls[1][0]).toEqual(new URL(`https://example.com/admin/models`))
+  expect(fauxFetch.mock.calls[1][0]).toEqual(new URL(`https://example.com/admin/modelData`))
```

Comment:
All the existing models methods now delegate to modelData. Between that and being deprecated, don't think tests are needed.

Reply:
So is there still any test or check that ensures this endpoint doesn't get removed from the codebase until we release a v3?

Reply:
If someone removed the endpoint, they'd remove the test as well. This is better handled with a note in the function documentation, with a link to a removal plan. Then code reviewers can see if it should be removed in a PR.

Reply:
If this is covered as part of the release process I think we can cover this sufficiently; deprecation warning is given and it should handle correctly with different client / server combinations.

@dbcfd added addt. notes to the Notion doc for this as well.

Reply:
FWIW I agree with Paul that in general as long as some code still exists and can be hit, deprecated or not, the tests for that code should remain.  It's not to prevent the code from being removed prematurely, it's to prevent the functionality from being accidentally broken while it's in the deprecated-but-not-yet-removed state.  As long as we want to support old clients hitting the old endpoint, we should keep the tests that ensure the old endpoint works as expected.

That said, not a huge deal and not worth going back to do anything about it at this point.